coderd/notifications/notifier.go

```diff
@@ -223,6 +229,18 @@ func (n *notifier) prepare(ctx context.Context, msg database.AcquireNotification
 		return nil, xerrors.Errorf("failed to resolve handler %q", msg.Method)
 	}
 
+	logoURL, err := n.store.GetLogoURL(ctx)
+	if err != nil && !errors.Is(err, sql.ErrNoRows) {
+		n.log.Error(ctx, "failed fetching logo url", slog.Error(err))
```

Comment:
Risk: We may end up showing the Coder logo here for a customer that requires it's never shown. It's better to not show a logo at all if this fails IMO.

Reply:
👍